canvas_zoom_meetings.py

```diff
@@ -0,0 +1,92 @@
+# Script to get all sites where Zoom is visible and retrieve the meetings to generate a report
+
+import json
+import re
+import os
+
+import requests
+import logging
+
+from canvasapi import Canvas
+from canvasapi.exceptions import ResourceDoesNotExist
+from collections import OrderedDict
+from bs4 import BeautifulSoup as bs
+from urllib.parse import urljoin
+
+# read configurations
+try:
+    with open(os.path.join('config', 'env.json')) as env_file:
+        ENV = json.loads(env_file.read())
+except FileNotFoundError:
+    logger.error(
+        'Configuration file could not be found; please add env.json to the config directory.')
+
+LOG_LEVEL = ENV.get('LOG_LEVEL', 'DEBUG')
+logging.basicConfig(level=LOG_LEVEL)
+
+# These two lines enable debugging at httplib level (requests->urllib3->http.client)
+# You will see the REQUEST, including HEADERS and DATA, and RESPONSE with HEADERS but without DATA.
+# The only thing missing will be the response.body which is not logged.
+import http.client as http_client
```

Comment:
move this up with other imports at the beginning of the file

Reply:
Done